src/main/java/core/basesyntax/services/FigureSupplier.java

```diff
@@ -0,0 +1,62 @@
+package core.basesyntax.services;
+
+import core.basesyntax.Color;
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import core.basesyntax.model.Figure;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int MAX_AMOUNT_OF_FIGURES = 6;
+    private static final int MAX_NUMBER = 10;
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        Circle defaultFigure = new Circle();
+        defaultFigure.setRadius(DEFAULT_RADIUS);
+        defaultFigure.setColor(Color.WHITE.name().toLowerCase());
+        return defaultFigure;
+    }
+
+    public Figure getRandomFigure() {
+        int figure = random.nextInt(random.nextInt(MAX_AMOUNT_OF_FIGURES) + 1);
```

Comment:
```suggestion
        int figure = random.nextInt(random.nextInt(MAX_AMOUNT_OF_FIGURES));
```

Reply:
When I do so, there appears a high probability of getting:

Exception in thread "main" java.lang.IllegalArgumentException: bound must be positive
	at java.base/java.util.Random.nextInt(Random.java:322)
	at core.basesyntax.services.FigureSupplier.getRandomFigure(FigureSupplier.java:28)
	at core.basesyntax.Application.main(Application.java:12)
	
I agree with all of your previous suggestions, but this one breaks the code.